@vmswr/vmswr.m

```diff
@@ -68,17 +68,21 @@
 	saveObject(obj,'ArgsC',Args);	
 else
 	rw = rplraw('auto',varargin{:});
-	if(~isempty(rw))		
-		[lpdata,resampleRate] = nptLowPassFilter(rw.data.analogData,rw.data.analogInfo.SampleRate, ...
+	if(~isempty(rw))
+        rlndata = nptRemoveLineNoise(rw.data.analogData, 50, rw.data.analogInfo.SampleRate); % remove 50Hz line noise
```

Comment:
I think we should make this an option that can be specified by the use of an optional argument instead of having this be run all the time.